### DISCUSSION TITLE
PMMA/MMA Combustion

### DISCUSSION THREAD

**Discussion by TristanHehnen** [firemodels/fds#12599] [Q&A]
Hi,

from FDS6.7.9 to FDS 6.8.0 the default soot definition changed. This leads to different heat of combustion values calculated by FDS. For the MMA in the  [NIST Parallel Panel validation case](https://github.com/firemodels/fds/blob/c11858054e384d5f64cad0c86be99d13c6334f00/Validation/NIST_NRC_Parallel_Panels/FDS_Input_Files/PMMA_60_kW_1_cm.fds#L12) (MaCFP reference/UMD), with a soot yield of 0.022g/g and a CO yield of 0.01g/g, this leads to:

24.3 MJ/kg, for FDS6.7.9-0-gec52dee-HEAD
28.8 MJ/kg, for FDS-6.8.0-124-g6542893-master

The SFPE Handbook (5th edition, table A.39) lists 24.2 MJ/kg with the same yields for CO and soot as implemented in the validation case.

[Zeng et al., 2002, Review on Chemical Reactions of Burning Poly(methyl methacrylate) PMMA](https://doi.org/10.1177/0734904102020005482) list 24.89 MJ/kg for the complete combustion of PMMA , this value is originally from [Huggett 1980, Estimation of rate of heat release by means of oxygen consumption measurements](https://doi.org/10.1002/fam.810040202) 

Zeng at al. also state that 
"In the absence of oxygen, thermal decomposition of PMMA is not affected by the environment (such as vacuum, nitrogen or argon), and the predominant product is monomer MMA (> 90%).". 
And further
"[...], there are other termination reactions that compete with the decomposition reaction. This is demonstrated by forming products such as CO, CO2, OH, CH4, and char [5,10–12], other than the monomer."

Why is the value of the heat of combustion from FDS now so much higher (~15%)? Could this be related to the ~10% other stuff than monomer, that is neglected here when the surrogate fuel is only MMA? 

Best,
Tristan

PS:
This may connect to the discussion in the [MaCFP Google Group](https://groups.google.com/g/macfp-discussions/c/S61GPR4ObjY).

**Comment by rmcdermo** [comment]
@TristanHehnen When you specify a species that is not pre-defined in FDS, then FDS has to back out the HOC from the heats of formation of the other reactants.  If you know the HOC, then you can simply add that as a parameter and FDS will change the enthalpy of formation to match.

**Comment by drjfloyd** [comment]
This input file really should have a HEAT_OF_COMBUSTION, and it probably should have had that even in 6.7.9. Assuming EPUMO2=13100 should be for when you have no other data.

6.8.0 introduced more effort to deal with detailed chemistry simulations and with that were changes to how REAC goes about figuring out what the HoC is when it isn't specified since we added more detailed tables of predefined species. Now if you have defined a reference enthalpy, that will be used unless overidden by HEAT_OF_COMBUSTION on REAC. By defining SPECIFIC_HEAT there is the implied REFERENCE_ENTHALPY of Cp*T_ref.  

With REFERENCE_ENTHALPY, ENTHALPY_OF_FORMATION, predefined species with tabulated properties, being able give enthalpy values for a primitive species or a species mixture, the logic gets quite convoluted to try and deal with all the use cases. It is a house of cards. This may need some better explanation in the Guide and maybe a WARNING message when a SPEC with just SPECIFIC_HEAT is used as a FUEL in a REAC with no HEAT_OF_COMBUSTION.

**Comment by drjfloyd** [comment]
I did the following:

-Added a WARNING when FDS will assume a reference enthalpy and no H_o_C or EPUMO2 is explicitly defined on REAC
-Modified the discussion of Enthalpy in Chapter 12 and Heat of Combustion in Chapter 13 to give more details on what FDS does
-Ensured that if a user does actually specify an EPUMO2 on REAC that it overrides any reference enthalpy or heat of formation associated on SPEC as described in the guide.

I ran the verification suite before comitting and everything was in tolerance. I will also edit the inputs for the PMMA cases to specify the H_o_C since it was measured as part of the MaCFP testing.

**Comment by TristanHehnen** [comment]
For clarification: as it was set up in 6.7.9, FDS did neither know the enthalpy of formation nor the heat of combustion of PMMA. Thus it assumed 13.1 KJ/kg for the heat of combustion. With the change of the soot, this changed the energy balance and the heat of combustion was coming out higher. Is this a correct understanding?

I've tried to use the `ENTHALPY_OF_FORMATION` with your new definition, instead of the `HEAT_OF_COMBUSTION`. The enthalpy of formation for MMA was taken from the [NIST Chemical WebBook](https://webbook.nist.gov/cgi/cbook.cgi?ID=C80626&Mask=28F/1000) to be -342.3 kJ/mol. With this, FDS (FDS-6.8.0-124-g6542893-master) computes a heat of combustion of 25073.7295 kJ/kg, which is just slightly off compared to the value you've set (24190 kJ/kg).

I've also used your new definition, releasing a slightly different gas composition. For both decomposition reaction steps, 90% release your MMA and the remaining 10% are either carbon dioxide or methane. This is to have two extremes based on the statements from [Zeng et al.](https://doi.org/10.1177/0734904102020005482). Then, I've computed an effective heat of combustion by dividing the total heat release by the total mass loss. 
This leads to:

21771.0 kJ/kg, for 10% carbon dioxide
26691.6 kJ/kg, for 10% methane

Are we somehow "lucky", that the effective heat of combustion of PMMA comes out to be close to the heat of combustion of MMA? 
Basically, does the composition of the 10% other stuff manage to produce some effective heat of combustion that happens to be close to the heat of combustion of MMA such that it does not "stand out"? 
I guess it does not matter too much for setups like the cone calorimeter and up, since everything blends together. It may have an impact for the micro-scale setups, if the other stuff can be localised to specific temperature intervals.

**Comment by drjfloyd** [comment]
We made changes to how we deal with enthalpy between 6.7.9 and 6.8.0 to better ensure conservation when solid materials pyrolyze into gases. In 6.7.9 specifying SPECIFIC_HEAT on SPEC didn't result in some assumed reference enthalpy that was used in the REAC. That changed in 6.8.0. If you had HEAT_OF_COMBUSITON on REAC, then that would override the assumption but the default assumed EPUMO2 did not -- an overlooked use case (it is as I noted earlier a convoluted set of logic to deal with all the ways a user can specificy SPEC and REAC.). As a result with 6.8.0 the &SPEC MMA line with SPECIFIC_HEAT was not using EPUMO2=13100. This resulted in the large shift in the HoC.  The soot and CO yield does change the ideal HoC a little, but not by that much.

I don't know so much that it is lucky that the HoC is stil close. You still have 90 % MMA so that is going to drive the HoC.

**Comment by TristanHehnen** [comment]
Okay, thank you both very much!
